doc/developing-the-website.md

```diff
@@ -11,3 +13,15 @@ That command will build the site and start a local server. To work on only the w
 Alternatively, to dynamically rebuild the site and refresh the browser _when changes happen_ (again, optionally with the `--nodocs` flag), run:
 
     node_modules/.bin/gulp watch
+
+### Using Make
+
+To build the website, run:
+
+    make build
```

Comment:
Doesn`t the normal build process have its own document? Or is this required to be able to `serve`?

Reply:
@janpio sorry Jan, I dint get your question. make build should build the website using makefile

Reply:
@janpio this process is not required at all. The Makefile simply mirrors what the Gulpfile does. I put the Makefile in the project when it was started because I wanted faster builds, but nobody except me liked using Make.

Reply:
@dblotsky  @janpio  @stevengill  should we go ahead and knock off the Makefile and its corresponding installation steps then?